front/src/views/user/eating/eating-new/eating-new.view.js

```diff
@@ -0,0 +1,171 @@
+            v.forEach(file => {
+              if (file.size > MAX_FILE_SIZE_SCHEDULET_PLACE_TO_EAT) {
+                valid = false;
+              }
+            });
+            return valid ? true : 'File size should be less than 10 MB!';
+          } else {
+            return true;
+          }
+        },
+        fileLength: v => {
+          if (v) {
+            return v.length <= 10 || 'Количество вложений не должно быть больше 10';
+          } else {
+            return true;
+          }
+        }
+      },
+      date: new Date().toISOString().substr(0, 10), // new Date().toLocaleDateString()
+      time: new Date().toTimeString().substr(0, 5),
+      notes: '',
+      link: '',
+      namePlace: '',
+      googlePlaceId: '',
+      lat: 0,
+      lng: 0,
+      files: new FormData(),
+      text: '',
+      snackbar: false,
+      color: ''
+    };
+  },
+  computed: {
+    google: gmapApi
+  },
+  methods: {
+    findPlace (place) {
+      this.googlePlaceId = place.place_id || '';
+      if (place.geometry) {
+        this.lat = place.geometry.location.lat();
+        this.lng = place.geometry.location.lng();
+      }
+    },
+    parseDate (date) {
+      if (!date) {
+        return null;
+      }
+
+      const [month, day, year] = date.split('/');
+      return `${year}-${month.padStart(2, '0')}-${day.padStart(2, '0')}`;
+    },
+    save () {
+      const payload = {
+        dateTime: `${this.date}T${this.time}`,
+        namePlace: this.namePlace,
+        notes: this.notes,
+        link: this.link,
+        googlePlaceId: this.googlePlaceId,
+        lat: this.lat,
+        lng: this.lng
+      };
+      if (this.$route.params.id) {
+        payload.id = Number(this.$route.params.id);
+        this.$store.dispatch('eating/updateEating', payload)
+          .then(() => {
+            this.text = this.$t('eatingNew.successfullyEdited');
+            this.snackbar = true;
+            this.color = 'success';
+            this.$store.dispatch('eating/uploadNewFilesEating', {
+              files: this.files,
+              eatingId: payload.id
+            });
+            setTimeout(() => this.$router.push({ name: 'ScheduleEatingPlace' }), 3000);
+          })
+          .catch(error => {
+            this.text = error;
+            this.snackbar = true;
+            this.color = 'error';
+          });
+      } else {
+        this.$store.dispatch('eating/createNewEating', payload)
+          .then(r => {
+            this.text = this.$t('eatingNew.successfullySaved');
+            this.snackbar = true;
+            this.$store.dispatch('eating/uploadNewFilesEating', {
+              files: this.files,
+              eatingId: r.data
+            });
+            setTimeout(() => this.$router.push({ name: 'ScheduleEatingPlace' }), 3000);
+          })
+          .catch(error => {
+            this.text = error;
+            this.snackbar = true;
+            this.color = 'error';
+          });
+      }
+    },
```

Comment:
do you realy need different text message? and do you really need different server controllers?

Reply:
not understand this comment